ta/pkcs11/src/attributes.c

```diff
@@ -207,6 +207,37 @@ bool get_bool(struct obj_attrs *head, uint32_t attribute)
 	return bbool;
 }
 
+bool attributes_match_reference(struct obj_attrs *candidate,
+				struct obj_attrs *ref)
+{
+	size_t count = ref->attrs_count;
+	unsigned char *ref_attr = ref->attrs;
+	uint32_t rc = PKCS11_CKR_GENERAL_ERROR;
+
+	if (!ref->attrs_count) {
+		DMSG("Empty reference: no match");
+		return false;
+	}
+
+	for (count = 0; count < ref->attrs_count; count++) {
+		struct pkcs11_attribute_head pkcs11_ref = { };
+		void *found = NULL;
```

Comment:
Could we rename `found` as `value` or so?

With name `found` I kinda first think it as boolean.

Reply:
agree

Reply:
Changed "found" to "value"